reinstall-cli/src/main.rs

```diff
@@ -0,0 +1,36 @@
+//! The main entrypoint for bootc-reinstall
+
+use anyhow::{ensure, Context, Result};
+use rustix::process::getuid;
+
+mod config;
+mod podman;
+mod prompt;
+pub(crate) mod users;
+
+const ROOT_KEY_MOUNT_POINT: &str = "/bootc_authorized_ssh_keys/root";
+
+fn run() -> Result<()> {
+    bootc_utils::initialize_tracing();
+
+    let config = config::ReinstallConfig::load().context("loading config")?;
+
+    tracing::trace!("starting bootc-reinstall");
+    // Rootless podman is not supported by bootc
+    ensure!(getuid().is_root(), "Must run as the root user");
+    let all_args = podman::command(&config.bootc_image, prompt::get_root_key()?);
+    println!("Going to run command: {}", all_args.join(" "));
+    prompt::temporary_developer_protection_prompt()?;
+    podman::run_podman(all_args)?;
```

Comment:
One thing that'd be cool I think is on failure, save all the provided interactive answers in e.g. `/run/bootc/reinstall.yaml` or so, and tell the user:

```
Installation answers saved in /run/bootc/reinstall.yaml
error: Out of disk space
```

Then when we re-run we detect that and offer to resume from saved answers.

Reply:
Followup in #1082